tools/download_boost.bat

```diff
@@ -0,0 +1,18 @@
+@echo off
+SET mydir=%~dp0
+SET curdir=%cd%
+
+CD %mydir%
+
+REM TODO: add progress bar, -q is quiet, if removing it the progress bar is in multiple lines
+ECHO Downloading Boost
+wget -q  https://dl.bintray.com/boostorg/release/1.68.0/source/boost_1_68_0.tar.gz
```

Comment:
I guess we should use curl and not wget for windows, wget is not supported out-of-box...

Reply:
Yes that would make sense to remove an extra step for users. I'll try it out with curl to make sure that would work ok and push a commit if so

Reply:
Fixed with 5646b0a